### PR TITLE
Console Demo Missing Include On Windows

### PR DESCRIPTION
**Pull request by rassweiler**
Missing #include <algorithm> in OISConsole.cpp.

### CODE REVIEW

**Comment by TheOnlyJoey** [comment]
Thanks for the pull request, merged!

File: demos/OISConsole.cpp [hunk 1]
```diff
@@ -1,3 +1,475 @@
+<<<<<<< HEAD
```
Comment (azmeuk) [inline demos/OISConsole.cpp]: It looks like there is some merging stuff left here.

File: demos/OISConsole.cpp [hunk 2]
```diff
@@ -1,3 +1,475 @@
...
+	
+	if( ReceiveNextEvent( 0, NULL, kEventDurationNoWait, true, &event ) == noErr )
+	{
+		SendEventToEventTarget(event, targetWindow);
+		std::cout << "Event : " << GetEventKind(event) << "\n";
+		ReleaseEvent(event);
+	}
+}
+#endif
+=======
```
Comment (azmeuk) [inline demos/OISConsole.cpp]: Here too, and line 942 too.

File: demos/OISConsole.cpp (hunk 1 above)
Comment (rassweiler) [inline demos/OISConsole.cpp]: Shit, I tried to update my copy cause I had pushed it to the wrong repo before. Sorry guys.

Comment (azmeuk) [inline demos/OISConsole.cpp]: I think this is indeed the right way to do :)

**Pull request merged**